cirq/google/xmon_device_test.py

```diff
@@ -55,6 +55,15 @@ def test_init():
         _ = d.duration_of(cirq.Gate().on(q00))
 
 
+@cirq.testing.only_test_in_python3
```

Comment:
Because there are no strings in the `__repr__` (which is what causes the problems) you can omit this decorator.